examples/notebook.ipynb

```diff
@@ -157,14 +277,64 @@
+      "\u001b[32m2024-12-19 12:57:24.696\u001b[0m | \u001b[34m\u001b[1mDEBUG   \u001b[0m | \u001b[36mnova.gateway\u001b[0m:\u001b[36msync_wrapper\u001b[0m:\u001b[36m50\u001b[0m - \u001b[34m\u001b[1mstream_move_to_trajectory_via_joint_ptp took 0.00 seconds\u001b[0m\n",
+      "\u001b[32m2024-12-19 12:57:24.697\u001b[0m | \u001b[1mINFO    \u001b[0m | \u001b[36mnova.gateway\u001b[0m:\u001b[36masync_wrapper\u001b[0m:\u001b[36m31\u001b[0m - \u001b[1mCalling execute_trajectory with args=('cell', <function move_forward.<locals>.movement_controller at 0x1183c7ec0>), kwargs={}\u001b[0m\n",
+      "\u001b[32m2024-12-19 12:57:27.802\u001b[0m | \u001b[1mINFO    \u001b[0m | \u001b[36mnova.gateway\u001b[0m:\u001b[36masync_wrapper\u001b[0m:\u001b[36m37\u001b[0m - \u001b[1mexecute_trajectory took 3.11 seconds\u001b[0m\n",
+      "\u001b[32m2024-12-19 12:57:27.803\u001b[0m | \u001b[1mINFO    \u001b[0m | \u001b[36mnova.gateway\u001b[0m:\u001b[36masync_wrapper\u001b[0m:\u001b[36m31\u001b[0m - \u001b[1mCalling deactivate_motion_group with args=(), kwargs={'cell': 'cell', 'motion_group': '0@ur'}\u001b[0m\n",
+      "\u001b[32m2024-12-19 12:57:27.871\u001b[0m | \u001b[1mINFO    \u001b[0m | \u001b[36mnova.gateway\u001b[0m:\u001b[36masync_wrapper\u001b[0m:\u001b[36m37\u001b[0m - \u001b[1mdeactivate_motion_group took 0.07 seconds\u001b[0m\n"
+     ]
+    }
+   ],
    "source": [
-    "async for state in my_robot.execute_motion_stream_async(motion=plan_result.motion, speed=25, response_rate_ms=500):\n",
-    "    current_location = state.current_location_on_trajectory\n",
-    "    print(f\"Current Location: {current_location}\")"
+    "from nova.core.movement_controller import move_forward\n",
+    "\n",
+    "async with controller[0] as motion_group:\n",
+    "    home_joints = (await motion_group.joints(\"Flange\")).joints\n",
+    "    current_pose = await motion_group.tcp_pose(\"Flange\")\n",
+    "    target_pose = current_pose @ Pose((1, 0, 0, 0, 0, 0))\n",
+    "\n",
+    "    actions = [\n",
+    "        jnt(home_joints),\n",
+    "        ptp(target_pose),\n",
+    "        jnt(home_joints),\n",
+    "        ptp(target_pose @ [200, 0, 0, 0, 0, 0]),\n",
+    "        jnt(home_joints),\n",
+    "        ptp(target_pose @ (300, 0, 0, 0, 0, 0)),\n",
+    "        jnt(home_joints),\n",
+    "        ptp(target_pose @ Pose((300, 0, 0, 0, 0, 0))),\n",
+    "        jnt(home_joints),\n",
+    "        ptp(target_pose @ Pose((400, 0, 0, 0, 0, 0))),\n",
+    "        jnt(home_joints),\n",
+    "        ptp(target_pose),\n",
+    "        jnt(home_joints),\n",
+    "    ]\n",
+    "\n",
+    "    await motion_group.run(actions, tcp=\"Flange\", movement_controller=move_forward)"
```

Comment:
I would prefer if the example took the planned motion and executes this directly. One could optimize the behaviour for planning several motions beforehand and executing them later one.